src/components/Tearsheet/Tearsheet.js

```diff
@@ -111,15 +114,18 @@ class Tearsheet extends Component {
               {...other}
             >
               {this.state.loading && (
-                <Loading className={`${namespace}__loading`}>
+                <LoadingMessage className={`${namespace}__loading`}>
                   <div className={`${namespace}__loading__message`}>
                     {this.state.loadingMessage}
                   </div>
-                </Loading>
+                </LoadingMessage>
               )}
 
               {renderSidebar && (
-                <section className={`${namespace}__sidebar`}>
+                <section
+                  aria-hidden={this.state.loading}
```

Comment:
Interesting! This indicates to screen readers that anything within these sections should not be read, right? If so, perhaps we can revisit programmatically 'skipping' the tab focus in combination with this to reduce the requirement on consumers to manage it themselves.

Reply:
@SimonFinney I'm not sure what you mean here. There's a difference between what `aria-hidden` is doing and what the rest of the code to prevent tab focus is doing. 

In addition, I don't believe we should try to mutate user-provided props... in this case, it would be mutating `renderMain` content.